### PR TITLE
[Kingston] Include waste data in CSV export.

### PR DESCRIPTION
**Pull request by dracos**
[skip changelog]

### CODE REVIEW

File: perllib/FixMyStreet/Reporting.pm [hunk 1]
```diff
@@ -123,6 +123,8 @@ sub construct_rs_filter {
         $where{"$table_name.state"} = [ FixMyStreet::DB::Result::Problem->fixed_states() ];
     } elsif ( $self->state ) {
         $where{"$table_name.state"} = $self->state;
+    } elsif ($self->cobrand->call_hook('dashboard_export_include_all_states')) {
+        # Do nothing, want all states
```
Comment (struan) [inline perllib/FixMyStreet/Reporting.pm:127]: Might be good to have a test that this works

**Review by struan — APPROVED** [review]
Two small things but looks good.

File: perllib/FixMyStreet/Cobrand/Kingston.pm [hunk 2]
```diff
@@ -792,4 +792,48 @@ sub admin_templates_external_status_code_hook {
...
+    $csv->add_csv_columns(
+        uprn => 'UPRN',
+        user_email => 'User Email',
+        user_phone => 'User Phone',
+        payment => 'Payment',
+        pro_rata => 'Pro rata payment',
+        admin_fee => 'Admin fee',
+        container => 'Subscription container',
+        current_bins => 'Bin count declared',
+        quantity => 'Subscription quantity',
```
[the remark this answers is not in the document]
Comment (struan) [reply perllib/FixMyStreet/Cobrand/Kingston.pm:810]: ah yes, whatever we show to the user for the payment reference is what we want I think.

File: perllib/FixMyStreet/Reporting.pm (hunk 1 above)
Comment (dracos) [reply perllib/FixMyStreet/Reporting.pm:128]: It did not. :) Due to the confirmed date range part of the query, which I've changed to created - probably better for waste anyway, given the DD reports that sit unconfirmed for weeks.

**Pull request merged**